config.go

```diff
@@ -193,6 +221,7 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		HypervisorPath:        hypervisor,
 		KernelPath:            kernel,
 		ImagePath:             image,
+		KernelParams:          kernelParams,
```

Comment:
The `cc-env` command should display the kernel params too. See commit 257b46a1f5e11637a712d9c796953a14fcc92286 for an example of adding extra details to the `cc-env` output (you'll need to update the `formatVersion` as that commit did).